install.sh

```diff
@@ -1,19 +1,36 @@
 #!/bin/bash
 
-# Backup old templates in case you have your custom templates
+# Copy templates
+read -r -p "Do you want to copy templates? [Y|n] " response
+if [[ $response =~ (y|yes|Y) ]];then
+  
+  # Backup old templates in case you have your custom templates
+  read -r -p "Do you want to backup your old templates? [Y|n] " response
 
-read -r -p "Do you want to backup your old templates? [Y|n] " response
+  if [[ ! $response =~ (n|no|N) ]];then
```

Comment:
why don't we just do `if [[ $response =~ (y|yes|Y) ]];then` as the same with other places. I might type `c` and it still save will probably feels a bit weird 🙏

Reply:
Or we can only accept `y|yes|Y` and `n|no|N` inputs, for other inputs we just re-ask the question? What do you think? 😁